### ISSUE TITLE
Legends are overlapping in Firefox

### ISSUE THREAD

**Issue by Manjunath-Siddappa** [masayuki0812/c3#1026]
Legends are overlapping in Firefox

i have attached screenshot .

![c3 js - legends](https://cloud.githubusercontent.com/assets/8860807/6389054/3d2daa62-bdc6-11e4-8e96-f383d9b53bba.jpg)

Please let me know how to resolve this

**Comment by ergo** [comment]
This also happens for me on random occasions, on next reload the legends can be rendered properly.

**Comment by Manjunath-Siddappa** [comment]
@ergo it is happening on first load itself.

i tried to reload it again, but same Result.

**Comment by ergo** [comment]
For me it happens on random.

**Comment by MrBuff** [comment]
Same Problem here. Any Ideas how to fix?

**Comment by mrj** [comment]
See issue #1005

**Comment by ribmo** [comment]
I'm now experiencing this issue on v4.10-rc1:

**Firefox:**
![screen shot 2015-03-04 at 09 12 12](https://cloud.githubusercontent.com/assets/7317426/6481011/71e7ae3a-c251-11e4-93ef-a2ebda6fc373.png)

**Chrome:**
![screen shot 2015-03-04 at 09 17 12](https://cloud.githubusercontent.com/assets/7317426/6481010/71e38396-c251-11e4-981b-6da0de751ef1.png)

**Comment by masayuki0812** [comment]
I'm sorry for the basic bug like this. Now it's been fixed in `v0.4.10-rc2` and I'll release this fix in `v0.4.10` after rc test.

**Comment by Manjunath-Siddappa** [comment]
Thank you

**Comment by LeMoussel** [comment]
Bug  may not been fixed in `v0.4.10` 

![legends are overlapping in firefox](https://cloud.githubusercontent.com/assets/1816573/13467395/57467d7e-e09e-11e5-9686-b0c5edc75183.png)

Test code : http://jsfiddle.net/nzw7449m/3/

Code :
```
var chart = c3.generate({
    data: {
        columns: [
            ['Country 1', 6, 5.95, 6.69, 7.47, 3.53, 0.92, 7.21, 4.02, 3.97, 4.18, 4.27],
            ['Country 2', 7.45, 7.31, 8.69, 8.74, 5.7, -6.15, 4.4, 4.67, 3.8, 2.02, 3.04],
            ['Country 3', 3.82, 3.06, 2.56, 4.43, 2.2, 1.23, 2.53, 2.4, 3.53, 2.62, 2.58],
            ['Country 4', 7.58, 8.57, 8.24, 7.54, 4.69, 3.68, 7.62, 6.55, 3.64, 4.02, 4.63],
            ['Country 5', 6.48, 5.78, 6.1, 6.51, 4.27, 1.15, 8.04, 4.57, 5.51, 4.99, 5.1]
        ],
    },
    legend: {
        show: true
    }
});
d3.selectAll("#chart .c3-legend-item").style("font-size","20px");
```

**Comment by LeMoussel** [comment]
Response on [StackOverflow](http://stackoverflow.com/a/35754635/710955), is due because font-size are setting after the chart render.One solution is to set the font size as regular CSS.
